src/System.Linq.Expressions/src/System/Runtime/CompilerServices/ReadOnlyCollectionBuilder.cs

```diff
@@ -456,21 +485,24 @@ public ReadOnlyCollection<T> ToReadOnlyCollection()
                 items = ToArray();
             }
             _items = Array.Empty<T>();
-            _size = 0;
+            Count = 0;
             _version++;
 
             return new TrueReadOnlyCollection<T>(items);
         }
 
+        // Ensures that the capacity of this list is at least the given minimum
+        // value. If the currect capacity of the list is less than min, the
+        // capacity is increased to twice the current capacity or to min,
+        // whichever is larger.
```

Comment:
Nit: It's better/neater to use an XML doc. That way, when people hover over the method its description will pop up.

```xml
<summary>
Ensures that the capacity of this list is at least the given minimum value.
</summary>
<param name="min">The minimum value to have allocated.</param>
<remarks>
If the current capacity of the list is less than <paramref name="min"/>,
the capacity is increased to the larger value of twice the current capacity
and <paramref name="min"/>.
</remarks>
private void EnsureCapacity(int min) { ... }
```